docs/cedarling/cedarling-quick-start.md

```diff
@@ -10,14 +10,189 @@ tags:
 
 # Cedarling Quick Start Guide
 
-## This content is in progress
+## Introduction
 
-The Janssen Project documentation is currently in development. Topic pages are being created in order of broadest relevance, and this page is coming in the near future.
+Cedarling is an authorization decision service that allows fine-grained access control using policies. 
```

Comment:
Authorization Decision Service (ADS) has emerged more recently to reflect authorization as a cloud-native service rather than a monolithic policy engine embedded in an application. So saying the "Cedarling is an authorization decision service" is a little misleading, and may discourage vendors from implementing an ADS using the Cedarling.